### ISSUE TITLE
Add mobile browsers to tests targets

### ISSUE THREAD

**Issue by FezVrasta** [FezVrasta/popper.js#138]
We should add mobile browsers such as Chrome mobile and Safari mobile to our automated SauceLabs tests.

I tried it in the past but the tests didn't work, the mobile browser opened `localhost` instead of the Karma runner URL.

If someone can setup a PR to add these browsers it would be cool. Or if anyone has informations on how to setup them.

**Comment by nadiam84** [comment]
Be careful with Android. Emulator for it has a **very** old Chrome version, something like 30. Which was released in the middle of 2013. And no way to update it.

**Comment by FezVrasta** [comment]
Thanks for the information! I guess we could start with Safari mobile and see what we can do to get an updated Chrome mobile...

**Comment by FezVrasta** [comment]
So, I ran the tests manually on iOS 9.3 and 10.1 and they mostly pass, there are 3 tests that have problems because of the way `vh` units work probably.

**Comment by FezVrasta** [comment]
I was able to setup tests for iOS 9.3, but I had to skip 3 tests on iOS because of the way Safari Mobile handles the viewport and the scrolling.

Sauce Labs recently released Android 6 and 7 support, I tried to run Android 6 but it uses the version 44 of Chrome, way too old and broken.
When I try Android 7 I get:

```
Google API devices are only supported for Se3 specifications: Android GoogleAPI Emulator
```

I opened a ticket with Sauce Labs support to understand what this error means.
